Author: Luca Boccassi <luca.boccassi@gmail.com>
Forwarded: https://github.com/systemd/systemd/pull/36891
Description: test: skip TEST-75-RESOLVED if resolved is disabled at build time
--- a/test/units/TEST-75-RESOLVED.sh
+++ b/test/units/TEST-75-RESOLVED.sh
@@ -16,6 +16,11 @@
 # shellcheck source=test/units/util.sh
 . "$(dirname "$0")"/util.sh

+if ! command -v resolvectl >/dev/null || ! command -v networkctl >/dev/null; then
+    echo "resolved/networkd not found, skipping..." | tee --append /skipped
+    exit 77
+fi
+
 # We need at least Knot 3.0 which support (among others) the ds-push directive
 if ! knotc -c /usr/lib/systemd/tests/testdata/knot-data/knot.conf conf-check; then
     echo "This test requires at least Knot 3.0. skipping..." | tee --append /skipped
